packages/time/__tests__/readable-semester.test.js

```diff
@@ -0,0 +1,19 @@
+// @ts-check
+
+import {expect} from 'chai';
+import {readableSemester} from '../lib/readable-semester.js';
+
+const assert = (semesterString, year, season) => {
```

Comment:
```suggestion
const assertSemester = (semesterString, year, season) => {
```

I try to avoid using `assert` since it's a nodejs primitive